### ISSUE TITLE
error trying to install locally

### ISSUE THREAD

**Issue by AndrewG-usr** [appbaseio/dejaVu#346]
I am trying to install dejavu locally i cloned it and when i try to run yarn i get the error : 

error source-map@0.7.3: The engine "node" is incompatible with this module. Expected version ">= 8". Got "6.17.1"
error Found incompatible module.

**Comment by lakhansamani** [comment]
@AndrewG-usr can you please try updating your node version?
It is recommended to have node version >= 8
https://github.com/appbaseio/dejavu/blob/dev/package.json#L11

**Comment by AndrewG-usr** [comment]
@lakhansamani   im sorry i am new to this how would i go about updating the node version?

**Comment by lakhansamani** [comment]
It depends on your Operating system.
You can install the latest version from here
https://nodejs.org/en/download/

**Comment by AndrewG-usr** [comment]
@lakhansamani thank you for your help. 

Can I run dejavu as a service?

**Comment by siddharthlatest** [comment]
@AndrewG-usr Yes, you can. If you are making it publicly available as-is, you don't need to do anything additional. 

If you are modifying the source, you need to [add an attribute to this project](https://softwareengineering.stackexchange.com/questions/218331/what-are-the-requirements-for-attribution-in-the-mit-license).

**Comment by eldyvoon** [comment]
Having the same issue, can't do ```npm i```

![image](https://user-images.githubusercontent.com/2796189/68001907-2a3bb100-fca1-11e9-9838-c9667cc9a0c2.png)

**Comment by lakhansamani** [comment]
@eldyvoon thank you for reporting this. I was able to replicate this with `npm i`.
It worked with `yarn` for me so can you try `yarn install`/ `yarn` to install the dependencies.

I will try to fix the issue with `npm i` soon.
